File: models/src/main/java/com/mdgd/pokemon/models/repo/schemas/Species.java
package com.mdgd.pokemon.models.repo.schemas;

import androidx.room.ColumnInfo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Species {

    @Expose
    @SerializedName("name")
    @ColumnInfo(name = "species_name")
    private String name;

    @Expose
    @SerializedName("url")
    @ColumnInfo(name = "species_url")
    private String url;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

}

File: models/src/main/java/com/mdgd/pokemon/models/repo/schemas/Species.kt
package com.mdgd.pokemon.models.repo.schemas

import androidx.room.ColumnInfo
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Species {
    @Expose
    @SerializedName("name")
    @ColumnInfo(name = "species_name")
    var name: String? = null

    @Expose
    @SerializedName("url")
    @ColumnInfo(name = "species_url")
    var url: String? = null
}
